src/db.ts

```diff
@@ -131,22 +138,34 @@ export default sbp('sbp/selectors/register', {
     try {
       const value: string = await sbp('chelonia.db/get', hash)
       if (!value) throw new Error(`no entry for ${hash}!`)
-      return SPMessage.deserialize(value, this.transientSecretKeys, undefined, this.config.unwrapMaybeEncryptedData)
+      return SPMessage.deserialize(
+        value,
+        this.transientSecretKeys,
+        undefined,
+        this.config.unwrapMaybeEncryptedData
+      )
     } catch (e) {
-      throw new ChelErrorDBConnection(`${(e as Error).name} during getEntry: ${(e as Error).message}`)
+      throw new ChelErrorDBConnection(
+        `${(e as Error).name} during getEntry: ${(e as Error).message}`
+      )
     }
   },
   'chelonia/db/addEntry': function (entry: SPMessage): Promise<string> {
     // because addEntry contains multiple awaits - we want to make sure it gets executed
     // "atomically" to minimize the chance of a contract fork
     return sbp('okTurtles.eventQueue/queueEvent', `chelonia/db/${entry.contractID()}`, [
-      'chelonia/private/db/addEntry', entry
+      'chelonia/private/db/addEntry',
+      entry
```

Comment:
I'd say the entire PR looks good to me, with this being the only eyebrow-raising change. Personally think it looked better before and see no reason to split it across lines like this.